### DISCUSSION TITLE
Route other containers through this container

### DISCUSSION THREAD

**Discussion by webepg** [haugene/docker-transmission-openvpn#1984] [Q&A] [answered]
Hi,

is it possible to route other containers through this container? If so, how would I do that?

thx

**Comment by aamirismail** [comment]
Hi,

Follow this.

https://www.youtube.com/watch?v=xbSfaKwyfXE&t=404s&ab_channel=DBTech
It works but its not stable as soon as your VPN container stops the other container using it jsut removes it as network and then you have to do the whole process again. 

I am looking for stable solution for this as well. 

At them moment i use separate vm for dockers which i need to connect throught vpn so vpn runs on host and all container use host network to reach out.

**Comment by pkishino** [comment]
You can use the proxy or look through the past issues, others have routed traffic through it

**Comment by pchristod** [comment, marked as answer]
One solution is to add the following to your external container. If you need any port mapping then you need to add that to your transmission container.
```
--net=container:containerName
```
